src/platform/pdf/test/templates/medications/fixtures/medications_list.json

```diff
@@ -39,7 +57,24 @@
                   "inline": true
                 },
                 {
-                  "value": "We’re processing a fill or refill for this prescription. We’ll update the status here when we ship your prescription."
+                  "isRich": true,
+                  "value": [
+                    {
+                      "value": "Your VA provider prescribed this medication or supply to you. But we won’t send any shipments until you request to fill or refill it."
+                    },
+                    {
+                      "value": "We may use this status for either of these reasons:"
+                    },
+                    {
+                      "value": [
+                        "We’re not sure when you’ll need to fill this prescription, or",
+                        "You told us you have too much of this medication or supply"
```

Comment:
@vmenshutin-bylight I'm assuming we don't need a period here?

Reply:
@agravell047 there was none before, it is a bulleted list